index.html

```diff
@@ -2891,6 +2891,40 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
+        </h3>
+        <p>
+          The <dfn id="member-iarc_rating_id"><code>iarc_rating_id</code> member</dfn>
+          is a <a>string</a> that represents an ID value of the IARC rating of the 
+          web application. It is intended to be used to determine which ages 
+          the web application is appropriate for.
```

Comment:
should we add that this is global thing, ie not a country specific thing... it maybe a bit of info in a note about IARC